examples/apps/echo/client/echo_client.py

```diff
@@ -1,401 +0,0 @@
-#! /usr/bin/env python3
```

Comment:
Please raise a separate PR to remove echo_client

Reply:
Done